samples/Nancy.Swagger.Annotations.Demo/Modules/GenericModule.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Collections.Generic;
+using Microsoft.Extensions.FileProviders;
+using Nancy.ModelBinding;
+using Nancy.Responses.Negotiation;
+using Nancy.Swagger.Demo.Models;
+using Nancy.Swagger.Annotations.Attributes;
+using Swagger.ObjectModel;
+using Nancy.Swagger.Services;
+
+namespace Nancy.Swagger.Demo.Modules
+{
+    public class GenericModule : NancyModule
+    {
+        public GenericModule(ISwaggerTagCatalog tagCatalog)
+            : base("/generic")
+        {
+            tagCatalog.AddTag(new Tag()
+            {
+                Description = "All response are wrapped in a base class that can include metadata",
+                Name = "Generics"
+            });
+
+            Get("/users", _ => GetUsers(), null, "GetUsers");
+
+            Post("/users", _ =>
+            {
+                var user = this.Bind<User>();
+                return PostUser(user);
+            });
+        }
+
+        [Route("GetUsers")]
+        [Route(HttpMethod.Get, "/users")]
+        [Route(Summary = "Get All Users")]
+        [SwaggerResponse(HttpStatusCode.OK, Message = "OK", Model = typeof(ApiResponse<IEnumerable<User>>))]
+        [Route(Tags = new[] { "Generics" })]
+        private ApiResponse<IEnumerable<User>> GetUsers()
+        {
+            return new ApiResponse<IEnumerable<User>>(new[] {new User {Name = "Vincent Vega", Age = 45}});
+        }
+
+        [Route(HttpMethod.Post, "/users")]
+        [Route(Summary = "Post New User")]
+        [SwaggerResponse(HttpStatusCode.OK, Message = "OK", Model = typeof(ApiResponse<User>))]
```

Comment:
You should also show off an example of multiple generics. You have you code for {Model}Of{Generic1}And{Generic2}, so you might as well present it.